corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -30,7 +30,9 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
         $modal.find('#js-confirmation-confirm').text(options.confirmText);
         $modal.find('#js-confirmation-cancel').text(options.cancelText);
 
-        $modal.find('#js-confirmation-confirm').click(function (e) {
+        var $confirmationButton = $modal.find('#js-confirmation-confirm');
+        $confirmationButton.off('click');
```

Comment:
Not sure if this is an issue but won't this not also remove that bootstrap handler that closes the dialog? Might it not be safer to use namespaces or keep a reference to the handler to remove later?

```javascript
$confirmationButton.off('.confirmationModal');
$confirmationButton.on('click.confirmationModal, function (e) {... });
```

Reply:
The dialog still closes, so `data-dismiss` must use a slightly different mechanism, but that's still a good point. Updated in a369d13e0e2